src/cli/commands/test/iac-local-execution/index.ts

```diff
@@ -66,7 +57,12 @@ export async function test(
     const iacOrgSettings = await getIacOrgSettings(orgPublicId);
 
     if (options.rules) {
-      await assertRulesFlagAvailable(orgPublicId, iacOrgSettings);
+      if (!iacOrgSettings.entitlements?.iacCustomRulesEntitlement) {
```

Comment:
nit: we could skip the nested if and have a single if. we can evaluate the options.rules first, and if it's false then it won't even do the second one.
e.g.
```suggestion
      if (options.rules && !iacOrgSettings.entitlements?.iacCustomRulesEntitlement) {
```